src/javascript/app/base/footer.js

```diff
@@ -28,16 +29,26 @@ const Footer = (() => {
     };
     
     const displayNotification = (message) => {
-        const $status_notification = $('#status_notification');
-        const $status_message_text = $('#status_notification_text');
-        const $close_icon = $('#status_notification_close');
-
-        $status_notification.css('display', 'flex');
-        $status_message_text.html(message);
+        const notification_storage = LocalStore.getObject('status_notification');
+        const time_difference = (window.time.unix() - notification_storage.close_time || 0);
```

Comment:
maybe worth waiting for `time` response, might get called before we have the response of time

Reply:
could you put `notification_storage.close_time || 0` in a bracket? also might be worth turning `notification_storage.close_time` into an integer